lib/systems/include/Controllers/LoadCellVectoringController.h

```diff
@@ -57,12 +57,20 @@ class TorqueControllerLoadCellVectoring : public virtual Controller
         writeout_.command = BaseControllerParams::TC_COMMAND_NO_TORQUE;
         writeout_.ready = false;
     }
+    /// @brief default contructor with balanced dfault values: rearTorqueScale = 1.0, regenTorqueScale = 1.0
```

Comment:
spelling: "default"

Reply:
What are regenTorqueScale and rearTorqueScale? How are they used? What makes this particular load cell-based torque vectoring different from our other controllers?

Reply:
look right above this at constructor comments